packages/html/src/plugins/responsive.ts

```diff
@@ -15,7 +15,7 @@ import {isImage} from "../utils/isImage";
  * | number[] A set of image sizes in pixels.
```

Comment:
Does the description of the steps param need updating?

Reply:
I think it's clear from the example. Do you think we should change this?

Reply:
Yes - I think this is confusing:

```
 * @param steps {number | number[]} The step size in pixels.
 * | number[] A set of image sizes in pixels.
```